bitnami/milvus/Chart.yaml

```
@@ -51,4 +51,4 @@ maintainers:
 name: milvus
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/milvus
-version: 15.0.8
+version: 15.1.0
```

Please bump patch version instead:
```suggestion
version: 15.0.9
```